### PR TITLE
Fixes for wine detection tutorial

### PR DESCRIPTION
**Pull request by larroy**
## Description ##

see tittle

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by stu1130** [comment]
@mxnet-label-bot add [pr-awaiting-review]
Thanks for your great work @larroy

**Comment by larroy** [comment]
https://github.com/apache/incubator-mxnet/issues/9094

**Comment by vandanavk** [comment]
@aaronmarkham for review/merge

**Comment by aaronmarkham** [comment]
I put the edits for MXNet installation on Raspberry Pi in this PR: #14172

**Comment by aaronmarkham** [comment]
I also just fixed the conflict from the new license header.

**Comment by ankkhedia** [comment]
@larroy Could you look into comments by @aaronmarkham

**Comment by aaronmarkham** [comment]
I updated the Pi instructions here: https://github.com/apache/incubator-mxnet/pull/14172
So @larroy, if you know the tutorial works, then LMK and I'll merge this update. I can raise my issues with the preprocessing part of the tutorial in a separate issue.

File: docs/tutorials/embedded/wine_detector.md [hunk 1]
```diff
@@ -100,13 +101,14 @@ with open('synset.txt', 'r') as f:
     synsets = [l.rstrip() for l in f]
 
 # Load the network parameters
-sym, arg_params, aux_params = mx.model.load_checkpoint('Inception_BN', 0)
+sym, arg_params, aux_params = mx.model.load_checkpoint('Inception-BN', 126)
```
Comment (szha) [inline docs/tutorials/embedded/wine_detector.md]: magic number?

**Comment by vandanavk** [comment]
@mxnet-label-bot update [Example, pr-awaiting-response]

**Comment by karan6181** [comment]
@larroy Could you please address the review comments? Thanks!

File: docs/tutorials/embedded/wine_detector.md (hunk 1 above)
Comment (wkcn) [reply docs/tutorials/embedded/wine_detector.md]: It is the number of epoch.

Comment (szha) [reply docs/tutorials/embedded/wine_detector.md]: Right, what I mean is this probably shouldn't be hard-coded?

Comment (wkcn) [reply docs/tutorials/embedded/wine_detector.md]: Agree.

**Comment by abhinavs95** [comment]
@larroy Could you please give an update on the PR? Thanks

**Comment by piyushghai** [comment]
@larroy Gentle ping...

**Comment by Roshrini** [comment]
@aaronmarkham @larroy Is this PR good to go?

**Comment by aaronmarkham** [comment]
@larroy has an [unresolved comment here](https://github.com/apache/incubator-mxnet/pull/13886#pullrequestreview-207628585). Once that's done and he's sure it works, I'm ok with the PR.

File: docs/tutorials/embedded/wine_detector.md (hunk 1 above)
Comment (larroy) [reply docs/tutorials/embedded/wine_detector.md]: Maybe, but if you follow the instruction, the model donloaded has that epoch for the checkpoint. What do you suggest instead?

**Comment by larroy** [comment]
@aaronmarkham I responded to the comment.

**Comment by larroy** [comment]
@mxnet-label-bot remove [pr-awaiting-response]

**Comment by larroy** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Pull request merged**